src/app/store/index.ts

```diff
@@ -2,7 +2,9 @@ import { isDevMode } from '@angular/core';
 import { MetaReducer } from '@ngrx/store';
 import { AppState } from './Types/states';
 
-export const retreatRadarStoreFeatureKey = 'retreatRadarStore';
+export const retreatRadarStoreUserFeatureKey = 'retreatRadarUserStore';
+export const retreatRadarStoreTripsFeatureKey = 'retreatRadarTripsStore';
+export const retreatRadarStoreEventsFeatureKey = 'retreatRadarEventsStore';
 
 const initialState = {};
```

Comment:
I can't see if this is being used anywhere, it's a const and not exported, so just double check if this is used